src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -1,91 +1,152 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import capitalize from 'lodash/capitalize';
 import moment from 'moment';
 
 import AsyncComponent from 'app/components/asyncComponent';
-import Button from 'app/components/button';
-import ButtonBar from 'app/components/buttonBar';
 import Card from 'app/components/card';
-import {HeaderTitle} from 'app/components/charts/styles';
+import ErrorPanel from 'app/components/charts/errorPanel';
+import OptionSelector from 'app/components/charts/optionSelector';
+import {
+  ChartControls,
+  HeaderTitle,
+  InlineContainer,
+  SectionValue,
+} from 'app/components/charts/styles';
 import LoadingIndicator from 'app/components/loadingIndicator';
 import {Panel, PanelBody} from 'app/components/panels';
 import QuestionTooltip from 'app/components/questionTooltip';
 import TextOverflow from 'app/components/textOverflow';
+import {IconCalendar, IconWarning} from 'app/icons';
 import {t, tct} from 'app/locale';
 import space from 'app/styles/space';
 import {DataCategory, Organization} from 'app/types';
 
-import Chart from './chart';
 import {OrganizationUsageStats} from './types';
+import UsageChart, {
+  CHART_OPTIONS_DATA_TRANSFORM,
+  CHART_OPTIONS_DATACATEGORY,
+  ChartDataTransform,
+  ChartStats,
+} from './usageChart';
 import {formatUsageWithUnits} from './utils';
 
 type Props = {
   organization: Organization;
   dataCategory: DataCategory;
   dataCategoryName: string;
+  dateStart: moment.Moment;
+  dateEnd: moment.Moment;
   onChangeDataCategory: (dataCategory: DataCategory) => void;
+  onChangeDateRange: (dateStart: moment.Moment, dateEnd: moment.Moment) => void;
 } & AsyncComponent['props'];
 
 type State = {
   orgStats: OrganizationUsageStats;
+  chartDataTransform: ChartDataTransform;
 } & AsyncComponent['state'];
 
 class UsageStatsOrganization extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      chartDataTransform: ChartDataTransform.CUMULATIVE,
+    };
+  }
+
+  /**
+   * Ignore this hard-coded method.
+   * This will be updated in a separate PR.
+   */
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
     const {organization} = this.props;
 
-    // TODO(org-stats): Allow user to pick date range
-    const fourWeeksAgo = moment().subtract(31, 'days').unix();
-    const today = moment().unix();
-
     return [
       [
         'orgStats',
         `/organizations/${organization.slug}/stats_v2/`,
         {
           query: {
-            start: fourWeeksAgo,
-            end: today,
             interval: '1d',
           },
         },
       ],
     ];
   }
 
+  get chartMetadata() {
+    const {orgStats} = this.state;
+
+    return {
+      ...this.mapStatsToChart(orgStats),
+    };
+  }
```

Comment:
This method may return more keys when `getEndpoints` return something is made, so I'm putting it in first.